src/wasm-validator.h

```diff
@@ -217,7 +217,7 @@ struct WasmValidator : public PostWalker<WasmValidator> {
 
   void validateAlignment(size_t align, WasmType type, Index bytes, bool isAtomic,
                          Expression* curr);
-  void validateMemBytes(uint8_t bytes, WasmType ty, Expression* curr);
+  void validateMemBytes(uint8_t bytes, WasmType type, Expression* curr);
```

Comment:
Heh... I seem to have absorbed LLVM's aversion to using 'type' in variable names.